src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -7020,6 +7056,53 @@ private void ApplyMemberPostConditions(BoundExpression? receiverOpt, MethodSymbo
             ApplyMemberPostConditions(receiverSlot, method);
         }
 
+        // For an instance method, or a non-static local function in an instance method, returns the slot for the `this` parameter
+        // For a static method, or a static local function in a static method, returns 0
+        // Otherwise, returns -1
+        private int GetReceiverSlotForMemberPostConditions(MethodSymbol? method)
+        {
+            if (method is null)
+            {
+                return -1;
+            }
+
+            bool isStatic = method.IsStatic;
+
+            MethodSymbol? current = method;
+            while (current is object)
+            {
+                if (current.IsStatic != isStatic)
+                {
+                    // If an instance local function is a static method or vice versa, 
```

Comment:
>an instance local function is a static method [](http://example.com/codeflow?start=26&length=45)

Does "instance local function" mean "non-static local function"? If so, let's use "non-static" instead to avoid implying there's an instance.

Reply:
"... **in** a static method"?